swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+class Clothing(Item):
+    def __init__(self, id = None, fabric = "Unknown", condition = 0, age = None):
+        super().__init__(id, condition, age)
+        self.fabric = fabric
+    def get_category(self):
+        return "Clothing"
```

Comment:
Instead of hardcoding a return value, "Clothing", we should think of a way to do this dynamically with code. 

What if `Clothing` changes to `Clothes`? Then we'd have to remember to update `get_category`. 

Luckily, Python allows us to get a class name: `self.__class__.__name__`

If you refactor `get_category` here in `Clothing`, `Decor`, `Electronics` you'd be repeating the same code in 3 child classes. If 3 classes have repeating code, it's a sign that refactoring should happen. 

In this scenario, OOP is how we can reduce repeated code and __share__ functionality across several classes. If the parent class has a `get_category` method that could dynamically return a class's name then you don't need to override it in all the child classes.